tests/Dataset.test.ts

```diff
@@ -94,14 +84,53 @@ describe("Dataset", () => {
     },
   };
 
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  const manifestV1_0_0: AnyManifestFile = {
+    ...manifestV0_0_0,
+    features: [
+      { name: "feature1", data: "feature1.json", units: "meters", type: "continuous" },
+      { name: "feature2", data: "feature2.json", units: "(m)", type: "discrete" },
+      { name: "feature3", data: "feature3.json", units: "μm/s", type: "bad-type" },
```

Comment:
Does `bad-type` represent an invalid or unsupported feature type? Do we have logic in the code to validate types or handle exceptions?

Reply:
Yes, `bad-type` is unsupported. There's a test in this file on line 127 that checks that bad/unrecognized types are defaulted to being `continuous` data!